docs/guide.md

```diff
@@ -0,0 +1,194 @@
+   Click OK and close the installer.
+8. Download Iris and Sodium from your website of choice:
+    - Iris Shaders: [Curseforge](https://www.curseforge.com/minecraft/mc-mods/irisshaders), [Modrinth](https://modrinth.com/mod/iris)  
+    - Sodium: [Curseforge](https://www.curseforge.com/minecraft/mc-mods/sodium), [Modrinth](https://modrinth.com/mod/sodium)
+9. Place the downloaded files (and optionally your other Fabric mods) into the mods folder. The location of the mods folder depends on  your operating system:
+    - Windows: `%APPDATA%\.minecraft\mods`  
+    - Linux: `~/.minecraft/mods`  
+    - MacOS: `~/Library/Application Support/minecraft/mods`
+10. Next time when you open the Minecraft launcher, you should have the following profile (the version numbers may be different):  
+    ![fabricprofile](guide-resources/fabricprofile.png)  
+    Play the game with this profile selected to run the game with Iris Shaders and the rest of your Fabric mods!
+    
+## Using Iris
+
+After you install Iris using one of the methods described in the [Installing Iris](#installing-iris) section, make sure that it is properly installed by going into the Minecraft video settings. The video settings menu should look like this:  
+
+![videosettings](guide-resources/videosettings.png)
+
+You probably noticed that the menu is completely different. It is a reworked video settings menu provided by Sodium. In the **General** and **Quality** tab, you can find all the usual settings from vanilla Minecraft which you can change to fit your preferences. The settings in the Performance and Advanced tabs should not be modified in most cases.
+
+Shader packs are managed in the **Shader packs** tab. The shader pack menu should look like this:  
+
+![shaderpacksmenu](guide-resources/shaderpacksmenu.png)
+
+This menu shows shader packs which are in the shader packs folder. You can add new shader packs by dragging and dropping them into this menu, or by clicking the "Open Shader Pack Folder..." button and adding the shader packs into the opened folder. To select a shader pack, click on "Shaders: Disabled" to enable them. After that, select the shader pack you want to enable by clicking on it, then enable the selected shader by clicking the Apply button. After setting up your shaders, click Done to go back to the video settings menu.
+
+After configuring the video settings, you can now load your world and play Minecraft with shaders! While in game, you can use **keyboard shortcuts** which allow you to quickly manage your shaders:  
+
+- Pressing "**O**" opens up the shader packs menu with a transparent background, so you can see changes happen in real time:  
+  ![menuingame](guide-resources/menuingame.jpg)
+- Pressing "**K**" toggles shaders on and off.
+- Pressing "**R**" reloads the currently active shader pack.
+
+You can rebind these keyboard shortcuts to any other key you want in the Controls menu.
+
+## Shader compatibility
```

Comment:
https://github.com/IrisShaders/Iris/pull/980 is adding a separate shader compatibility list file, once it is merged I would like for this section to link to that instead of having a bunch of different out-of-sync shader compatibility lists.

Reply:
Sure thing, will update the PR as soon as the file is merged

Reply:
Should be merged now.

Reply:
updated